github/github_test.go

```diff
@@ -674,6 +674,40 @@ func TestDo_rateLimit_abuseRateLimitError(t *testing.T) {
 	}
 }
 
+// Ensure *AbuseRateLimitError is returned when the response indicates that
+// the client has triggered an abuse detection mechanism on github enterprise.
```

Comment:
I believe Glenn was referring to the string here, which can be made "GitHub Enterprise".